.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,9 @@
-* [ ] ❗ I have followed the [Contributing to DVCLive](https://github.com/iterative/dvclive/blob/master/CONTRIBUTING.md) guide.
```

Comment:
I think those could be excluded btw. As far as I remember formatting them breaks the rendering in some subtle way. We are excluding them in all the websites.

Reply:
https://github.com/executablebooks/mdformat/issues/359 😅